apps/linear_algebra/benchmarks/CMakeLists.txt

```diff
@@ -34,19 +34,25 @@ target_compile_definitions(halide_benchmarks PRIVATE -DUSE_HALIDE ${MISC_DEFINIT
 target_link_libraries(halide_benchmarks PRIVATE halide_blas)
 list(APPEND BLAS_NAMES halide)
 
-find_package(Eigen3 QUIET)
-if (NOT Eigen3_FOUND)
+find_package(PkgConfig)
+if (PkgConfig-NOTFOUND)
+  find_package(Eigen3 QUIET)
+  set(EIGEN3_FOUND NOT Eigen3-NOTFOUND)
+  if (EIGEN3_FOUND)
+    set(EIGEN3_LDFLAGS Eigen3::Eigen)
+  endif()
+else()
+  pkg_check_modules(EIGEN3 eigen3)
```

Comment:
Use IMPORTED_TARGET here... https://cmake.org/cmake/help/v3.14/module/FindPkgConfig.html#command:pkg_check_modules

Reply:
I didn't fully understand the IMPORTED keywords, so hopefully it won't be relevant after the next commit.